docs/development/workflow/development_workflow.rst

```diff
@@ -433,24 +433,42 @@ to GitHub. GitHub will automatically update your pull request.
 Rebase, but only if asked
 =========================
 
-Sometimes the maintainers of Astropy will ask you to *rebase* your changes
-before they are merged into the main Astropy repository.
+Sometimes the maintainers of Astropy may ask a pull request to be *rebased*
+(`git rebase`) or *squashed*/*squished* (`git merge --squash`) before
```

Comment:
1. Use double backticks (also apply to all other single backticks on command line examples
```
``git rebase``
```

2. I have never seen anyone write "squish the commits" but I guess it doesn't hurt...